Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -1139,6 +1139,9 @@
     <PackageReference Include="Microsoft.CSharp">
       <Version>4.5.0</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.Internal.VisualStudio.Shell.Interop.14.0.DesignTime">
+      <Version>14.0.25023-setup</Version>
```

Comment:
I'm kind of surprised there's a suffix on the version number.  For the LS, we're just using `14.0.0`.

Reply:
Fixed!